### PR TITLE
Introduced translation architecture using KF5I18N

### PR DESCRIPTION
**Pull request by shalomRachapudi**
Please have a look at po/CMakeLists.txt, .pot and .po files as well. 
i18n* calls are in my next pr. Thank you :)

### CODE REVIEW

File: po/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+find_package(KF5I18n)
+set(LOCALE_INSTALL_DIR /usr/share/locale/)
```
Comment (antlarr) [inline po/CMakeLists.txt]: is LOCALE_INSTALL_DIR needed?  What happens if this line is removed?

Comment (shalomRachapudi) [inline po/CMakeLists.txt]: It is by default SET to /share/locale
So, if that line is removed, mo files will be installed at share/locale/xx/LC_MESSAGES/ which is not the correct path to connect catalogs.

Comment (antlarr) [inline po/CMakeLists.txt]: are you sure of that? are you using cmake -DCMAKE_INSTALL_PREFIX=/usr .. ?

Comment (shalomRachapudi) [inline po/CMakeLists.txt]: Nope. I thought that this would make things easy :/

Comment (shalomRachapudi) [inline po/CMakeLists.txt]: I have deleted that line. Will update readme.md accordingly in my next pr

**Pull request merged**